.env

```diff
@@ -0,0 +1,5 @@
+REACT_APP_IMAGE_URL="https://apps.perseids.org/woodhouse/android-chrome-512x512.png"
+REACT_APP_SITE_NAME="Woodhouse English-Greek Dictionary"
+REACT_APP_TITLE="Woodhouse English-Greek Dictionary"
+REACT_APP_DESCRIPTION="This is the entire English-Greek Dictionary: A Vocabulary of the Attic Language by S. C. Woodhouse "
+REACT_APP_URL="https://perseids-project.github.io/woodhouse-js/"
```

Comment:
It looks like there's a newline missing here (that's what that weird looking symbol at the end of the line means). Could you add a newline and push the changes?